Dockerfile

```diff
@@ -17,7 +17,6 @@ RUN bundle install
 
 ADD . $APP_HOME
 
-ARG COMPILE_ASSETS=false
-RUN if [ "$COMPILE_ASSETS" = "true" ] ; then bundle exec rails assets:precompile ; fi
+RUN bundle exec rails assets:precompile
```

Comment:
I just had a look at this and this will produce a different outcome dependent on the RAILS_ENV. Annoyingly it fails if we have `RAILS_ENV=production bundle exec rails assets:precompile` due to a missing env var though. 

Probably be good to work out a nice way to get this to be sorted before doing lots of the PRs

Reply:
might be fine if we just do it as `GOVUK_APP_DOMAIN=www.gov.uk RAILS_ENV=production bundle exec rails assets:precompile` bit hacky though :-/ - it fails due to this: https://github.com/alphagov/calendars/blob/master/config/environments/production.rb#L73